Source/Core/Common/Timer.cpp

```diff
@@ -203,14 +205,19 @@ std::string Timer::GetTimeFormatted()
 
   struct tm* gmTime = localtime(&sysTime);
 
+#ifdef _WIN32
```

Comment:
Isn't this something that:

```cpp
std::ostringstream os;
os << std::put_time(gmTime, "%M:%S);
```
can avoid needing the ifdef here, since it uses the underlying locale?

Reply:
We can't be dealing with non-wide strings when using any locale other than the classic C locale, otherwise we don't have any simple way to convert the result (which is in some kind of Windows code page encoding) into UTF-8. Both strftime and your suggestion have this problem as far as I can tell.

Reply:
oh, ok, disregard the above suggestion then.